pom.xml

```diff
@@ -59,6 +52,25 @@
             <artifactId>lombok</artifactId>
             <optional>true</optional>
         </dependency>
+
+<!--        <dependency>-->
+<!--            <groupId>org.codehaus.groovy</groupId>-->
+<!--            <artifactId>http-builder</artifactId>-->
+<!--        </dependency>-->
+
+<!--        <dependency>-->
+<!--            <groupId>org.codehaus.groovy</groupId>-->
+<!--            <artifactId>groovy-eclipse-compiler</artifactId>-->
+<!--            <version>3.3.0-01</version>-->
+<!--        </dependency>-->
+
+<!--        <dependency>-->
+<!--            <groupId>org.codehaus.groovy</groupId>-->
+<!--            <artifactId>groovy-eclipse-batch</artifactId>-->
+<!--            <version>2.4.17-01</version>-->
+<!--        </dependency>-->
+
+
```

Comment:
what are these comments for?

Reply:
I have already told you the reason in telegram, and i have also commented on the pull request to not merge it. just pull the branch and starts working on it. i could not write a functional test. maybe you can try write a functional test for login

Reply:
I have already told you the reason in telegram, and i have also commented on the pull request to not merge it. just pull the branch and starts working on it. i could not write a functional test. maybe you can try write a functional test for login

Reply:
I didn't understand that, paste it one more time :)))